### PR TITLE
Fixes being able to use certain abilities while stunned.

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

File: code/modules/spells/spell_types/turf_teleport.dm [hunk 1]
```diff
@@ -12,6 +12,10 @@
 	var/sound2 = "sound/weapons/ZapBang.ogg"
 
 /obj/effect/proc_holder/spell/targeted/turf_teleport/cast(list/targets,mob/user = usr)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.stunned || H.weakened || H.sleeping || H.stat)
```
Comment (ChangelingRain) [inline code/modules/spells/spell_types/turf_teleport.dm]: half your checks do nothing you butt

Comment (Iamgoofball) [inline code/modules/spells/spell_types/turf_teleport.dm]: yes they do???

**Comment by Incoming5643** [comment]
N E V E R  N E R F  W I Z A R D S

File: code/modules/spells/spell_types/turf_teleport.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/spells/spell_types/turf_teleport.dm]: no they don't you butt read all of spell code

**Pull request closed without merging**